docs/source/TQNet_Machine_Learning/Pre-Defined_Model_Architectures.md

```diff
@@ -114,3 +114,123 @@ The Hybrid Quantum Long Short-Term Memory (HQLSTM) is an advanced version of the
 - **optim:** String (Default: 'Adam') | Options: ['Adam', 'AdamW', 'SGD'] | The optimization algorithm used for training.
 - **loss_func:** String (Default: 'MSE') | Options: ['MSE', 'MAE'] | The loss function used to evaluate the performance of the model.
 - **target_column:** String (Default: 'Power, kW') | The target column that the model should learn.
+
+  
+### Loss functions  
+A loss function is a function that compares the output values predicted by the network with the target values provided by the user.   
+It estimates how well the neural network models the training data and during the training phase, the aim is to minimise the loss between the expected and predicted output values.  
+In supervised learning, the loss functions are classified into two classes according to the type of learning task: regression and classification loss functions  
+  
+##### Regression
+A regression loss function predicts a continuous quantity  
+  
+###### Mean Square Error (MSE)  
```

Comment:
Missing a `d` it should be `Mean Squared Error`, right?

Reply:
sorry for the typo.
added it